html/apps/changeUserRoles/js/index.js

```diff
@@ -12,10 +11,35 @@
 
   $html.on('click.ui.dropdown', '.js-dropdown [data-dropdown-value]', function(e) {
     e.preventDefault();
-    var $item = $(this);
-    var $dropdown = $item.parents('.js-dropdown');
-    $dropdown.find('.js-dropdown__input').val($item.data('dropdown-value'));
-    $dropdown.find('.js-dropdown__current').text($item.text());
+    var item = $(this);
+    var role = item.data("dropdown-value");
+    var email = item.closest('.c-dropdown').data('email').trim();
+    var userRole = item.text();
+    var $dropdown = item.parents('.js-dropdown');
++    $dropdown.find('.js-dropdown__input').val(item.data('dropdown-value'));
++    $dropdown.find('.js-dropdown__current').text(item.text());
+
+    var endPoint = "/aaa/changeRoles";// the endpoint to post data
+    // console.log(item.parent().text());
+    console.log(item.parent().text());
+    item.parent().text(item.text());
+
+
+       $.ajax({
+           type: "POST",
+           url: endPoint,
+           data: "user="+email+"&role="+role,
+           processData: false,
+           contentType: 'application/x-www-form-urlencoded',
+           complete: function (xhr) {
+               if (xhr.status == 200) {
+                   alert("Role changed successfully");
```

Comment:
How about using [toastr](http://codeseven.github.io/toastr/demo.html) or something else than `alert` to inform the use. But of course this is personal opinion and just a proposal :smile:. I do not like focus stealing of `alert()` and using alert is okay for the beginning.